plugins/package-managers/conan/src/main/kotlin/ConanV2Handler.kt

```diff
@@ -102,8 +103,17 @@ internal class ConanV2Handler(private val conan: Conan) : ConanVersionHandler {
             name = SCOPE_NAME_DEV_DEPENDENCIES,
             dependencies = parseDependencyTree(pkgInfosV2, projectInfo.buildRequires, workingDir)
         )
+        val testDependencies = parseDependencyTree(pkgInfosV2, projectInfo.testRequires, workingDir)
+        val testDependenciesScope = if (testDependencies.isNotEmpty()) {
+            Scope(
+                name = SCOPE_NAME_TEST_DEPENDENCIES,
+                dependencies = parseDependencyTree(pkgInfosV2, projectInfo.testRequires, workingDir)
+            )
+        } else {
+            null
+        }
```

Comment:
Maybe instead:

        val testDependenciesScope = testDependencies.takeIf { it.isNotEmpty() }?.let {
            Scope(
                name = SCOPE_NAME_TEST_DEPENDENCIES,
                dependencies = parseDependencyTree(pkgInfosV2, projectInfo.testRequires, workingDir)
            )
        }